commons/src/main/java/fr/free/nrw/commons/upload/VolleyRequestQueue.java

```diff
@@ -0,0 +1,42 @@
+package fr.free.nrw.commons.upload;
+
+import android.content.Context;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.toolbox.ImageLoader;
+import com.android.volley.toolbox.Volley;
+
+
+public class VolleyRequestQueue {
+    private static VolleyRequestQueue mInstance;
+    private RequestQueue mRequestQueue;
+    private ImageLoader mImageLoader;
+    private static Context mContext;
```

Comment:
This is only needed to initialize the VolleyRequestQueue and does not need to be retained